airbyte-webapp/src/packages/cloud/views/layout/MainView/MainView.tsx

```diff
@@ -62,8 +68,13 @@ const MainView: React.FC<React.PropsWithChildren<unknown>> = (props) => {
     <div className={styles.mainContainer}>
       <InsufficientPermissionsErrorBoundary errorComponent={<StartOverErrorView />}>
         <SideBar />
-        <div className={classNames(styles.content, { [styles.alertBanner]: !!alertToShow })}>
-          {alertToShow && <AlertBanner message={alertMessage} />}
+        <div
+          className={classNames(styles.content, {
+            [styles.alertBanner]: !!alertToShow,
```

Comment:
```suggestion
            [styles.alertBanner]: !!alertToShow && !showExperimentBanner,
```

Since otherwise we'll apply both if the speedyconnection banner is shown, which is not what we want here, since at that point it depends on the order of the CSS rules in the CSS files which apply, which is a "prioritizatin" we should never use (since we also have order of rules handled automatically by stylelint).